src/OfflineSearch.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.OfflineSearch.OfflineSearch" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Offline Search</name>
+    <description>Search Mendix List views, Template Grids and Data Grids using a single-field search</description>
+    <icon />
+
+    <properties>
+        <property key="targetGridName" type="string" required="true" defaultValue="">
+            <caption>Grid name</caption>
+            <category>Data source</category>
+            <description>The name of the grid on this page</description>
+        </property>
```

Comment:
This entire property implies support of only grids (template & data)... Perhaps use a more neutral name.